core/domain/state_domain.py

```diff
@@ -2418,21 +2288,21 @@ def get_translatable_contents_collection(
 
         subtitled_htmls = self.get_subtitled_html()
         for subtitled_html in subtitled_htmls:
-            html_string = subtitled_html.html
-            # Make sure we don't include content that only consists of
```

Comment:
Was this accounted for anywhere?

Reply:
Yes, I had discussed this in the team group chat and have included the same here:   https://github.com/oppia/oppia/blob/72acd1b75cbe599ede6eae760bacbfa1a40090db/core/domain/translation_domain.py#L359